homeassistant/components/nederlandse_spoorwegen/sensor.py

```diff
@@ -165,14 +169,29 @@ def device_state_attributes(self):
     @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def update(self):
         """Get the trip information."""
+
+        # If looking for a specific trip time, update around that trip time only.
+        if self._time and (
+            (datetime.now() + timedelta(minutes=30)).time() < self._time
```

Comment:
`datetime` instances are natively comparable. We don't need to call `.time()`.

https://docs.python.org/3/library/datetime.html#datetime-objects

Reply:
But `datetime.now() + timedelta(minutes=30)` is a datetime.**datetime** format and `self._time` is a datetime.**time** format. Without `time()` I get an error that comparison is not possible between these formats. Do you see another solution?

Reply:
Sorry, you're correct.